scripts/st2bootstrap-deb.sh

```diff
@@ -165,8 +176,22 @@ get_full_pkg_versions() {
 install_st2() {
   # Following script adds a repo file, registers gpg key and runs apt-get update
   curl -s https://packagecloud.io/install/repositories/StackStorm/${REPO_PREFIX}${RELEASE}/script.deb.sh | sudo bash
-  STEP="Get package versions" && get_full_pkg_versions && STEP="Install st2"
-  sudo apt-get install -y st2${ST2_PKG_VERSION}
+
+  if [ "$DEV_BUILD" = '' ]; then
+    STEP="Get package versions" && get_full_pkg_versions && STEP="Install st2"
+    sudo apt-get install -y st2${ST2_PKG_VERSION}
+  else
+    curl -SsL -k -o ./jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
+    chmod +x ./jq
+    PACKAGE_URL="$(curl -Ss -q https://circleci.com/api/v1.1/project/github/StackStorm/st2-packages/${DEV_BUILD}/artifacts | ./jq -r '.[] | select(.path | test("/home/ubuntu/packages/${SUBTYPE}/st2_.*.deb"; "i")) | .url')"
```

Comment:
Spent good portion of the day trying to make this line to resolve ${SUBTYPE} properly in Xenial. Unsuccessfully. 

Bash black magic is killing me.